src/lib/serviceRoutinesV2/postEntity.cpp

```diff
@@ -100,14 +100,14 @@ std::string postEntity
     }
   }
 
-  // Default value for status code: SccCreated
+  // Default value for status code: SccNoContent
   if ((ciP->httpStatusCode == SccOk) || (ciP->httpStatusCode == SccNone) || (ciP->httpStatusCode == SccCreated))
```

Comment:
So, if that 'if' evaluates to TRUE, then ciP->httpStatusCode is set to SccNoContent and HTTP header Location is added.
Do we want Location for postEntity?

Seems more like postEntities to me ...

What is clear is that (ciP->httpStatusCode == SccCreated) must be removed or replaced with (ciP->httpStatusCode == SccNoContent).

If removed or replaced, well that depends on whether any function sets ciP->httpStatusCode to SccNoContent before this moment in execution. I would keep it, to be safe